_posts/en/newsletters/2024-09-13-newsletter.md

```diff
@@ -0,0 +1,122 @@
+  any DLC, the price oracles are prevented from learning the contract
+  details or even that their price information is being used in a
+  contract.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+FIXME:stickies-v
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/30352#l-18FIXME"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.18.3-beta.rc2][] is a release candidate for a minor bug fix
+  release of this popular LN node implementation.
+
+- [BDK 1.0.0-beta.2][] is a release candidate for this library for
+  building wallets and other Bitcoin-enabled applications.  The original
+  `bdk` Rust crate has been renamed to `bdk_wallet` and lower layer
+  modules have been extracted into their own crates, including
+  `bdk_chain`, `bdk_electrum`, `bdk_esplora`, and `bdk_bitcoind_rpc`.
+  The `bdk_wallet` crate "is the first version to offer a stable 1.0.0 API."
+
+- [Bitcoin Core 28.0rc1][] is a release candidate for the next major
+  version of the predominant full node implementation.  A [testing
+  guide][bcc testing] is available.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+_Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch, so those changes will likely not be released until
+about six months after the release of the upcoming version 28._
+
+- [Bitcoin Core #30509][] adds an `-ipcbind` option to `bitcoin-node` to allow
+  other processes  to connect to and control the node via a unix socket. In
+  combination with the upcoming PR [Bitcoin Core #30510][], this will allow an
+  external [Stratum v2][topic pooled mining] mining service to create, manage,
+  and submit block templates. This is part of the Bitcoin Core [multiprocess
+  project][multiprocess project]. See Newsletters [#99][news99 multi] and
+  [#147][news147 multi].
+
+- [Bitcoin Core #29605][] changes peer discovery to prioritize peers from the
+  local address manager over fetching from seed nodes, to reduce the influence
```

Comment:
I think it could be beneficial to have an additional sentence explaining how, before this PR, seed nodes can influence peer selection and cause unnecessary information sharing